tests/integration/test_users.py

```diff
@@ -81,21 +81,52 @@ def test_refresh_user(client, test_database, admin_user):
     assert responseRefreshToken.status_code == 200
 
 
-def test_get_user_by_id(client, auth_headers, admin_user):
+def test_get_user_by_id(client, empty_test_db, create_admin_user, admin_header):
     """The get user by id route returns a successful response code."""
-    user = UserModel.find_by_email(admin_user.email)
-    response = client.get(f"/api/user/{user.id}", headers=auth_headers["admin"])
+    user = create_admin_user()
+    response = client.get(f"/api/user/{user.id}", headers=admin_header)
     assert response.status_code == 200
 
     """
     The server responds with an error if a non-existent user id
     is requested from the get user by id route.
     """
-    responseBadUserId = client.get("/api/user/000000", headers=auth_headers["admin"])
+    responseBadUserId = client.get("/api/user/000000", headers=admin_header)
     assert responseBadUserId.status_code == 404
     assert responseBadUserId.json == {"message": "User not found"}
 
 
+def test_get_pm_by_id(
+    client,
+    empty_test_db,
+    create_property_manager,
+    create_property,
+    create_lease,
+    admin_header,
+):
+    user = create_property_manager()
+    prop = create_property(manager_ids=[user.id])
+    lease_1 = create_lease(property=prop)
+    lease_2 = create_lease(property=prop)
+
+    response = client.get(f"/api/user/{user.id}", headers=admin_header)
+
+    property_list = response.json["properties"]
+    tenants_list = response.json["tenants"]
+
+    """The get user by id route returns a successful response code for PMs"""
+    assert response.status_code == 200
+
+    """The PM's properties are returned as a list of JSON objects"""
+    assert property_list == [prop.json()]
+
+    """
+    Tenants are retreived through the leases on each
+    property and returned as a list of JSON objects
+    """
+    assert tenants_list == [lease_1.tenant.json(), lease_2.tenant.json()]
```

Comment:
This is good! I like it. I'd probably want to expand on this to see the whole structure of the response. Depending on if we have tests for the json methods we can do something more or less like.

`assert request.json == user.json()`

Of course that would mean that the serialization in the endpoint needs to be moved to the json method, and that json method should be unit tested.

IF the serialization remains in the endpoint then I would like to see the other values asserted as well.

something like this:

```
assert request.json == {
  "name": user.name
  "phone": user.phone
  ...
  "properties": [prop.json()]
  "tenants": [lease_1.tenant.json(), lease_2.tenant.json()]
}
```

Reply:
You bring up a great point here, and in your comment about having the PM's `tenants` serialization moved into the json method.  

If you're okay with it, I'd like to address both of these in a separate issue/PR, just so we get something that works merged quickly since right now that functionality is broken in the frontend.

Reply:
Ok, but this shouldn't be holding up the frontend work.

Reply:
I'm fine with doing this in a separate or though